articles/healthcare-apis/fhir/use-smart-on-fhir-proxy.md

```diff
@@ -35,6 +35,17 @@ After you complete these steps, you should have:
 - A FHIR server with rge audience set to `https://MYFHIRAPI.azurehealthcareapis.com`, where `MYFHIRAPI` is the name of your Azure API for FHIR instance.
 - A public client application registration. Make a note of the application ID for this client application.
 
+### Configure Admin Consent for your app
+SMART on FHIR will require you to go through the process of authenticating and authorising the app. The first time will also require you to get an admin consent to let the app access your FHIR resources. If you don't have an ownership role over the app, you will have to reach out to the owner and ask them to click on the admin consent. In the case that you do have administrative priviliges, you should add yourself as an owner of the app, in order to approve the consent once.
```

Comment:
I'd suggest that we change the last part and fix the typo: In the case that you do have administrative privileges, you can grant the admin consent to yourself directly here, or do so later when you are prompted in the app. 

You can also add users as owners so they can view and edit this application registration.